### PR TITLE
Remove deprecated SCL versions

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request removes deprecated SCL versions.

### CODE REVIEW

**Comment by hhorak** [comment]
LGTM

**Comment by bamachrn** [comment]
#dotests

File: index.d/centos.yaml [hunk 1]
```diff
@@ -1167,36 +1022,12 @@ Projects:
...
-    git-path: 6/
-    target-file: Dockerfile
-    desired-tag: latest
-    notify-email: hhorak@redhat.com
-    build-context: ./
-    depends-on: centos/s2i-base-centos7:latest
-
   - id: 144
     app-id: centos
     job-id: nodejs-8-centos7
```
Comment (bamachrn) [inline index.d/centos.yaml]: could you please update `nodejs-8-centos7` to `nodejs-10-centos7`?

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
ok to test

**Comment by bamachrn** [comment]
Hi FYI, we are facing few issues due to the centos8 image inclusions, so builds are paused right now. you will receive emails as the builds are started again.

**Pull request merged**